framework/source/class/qx/ui/form/VirtualSelectBox.js

```diff
@@ -519,26 +517,40 @@ qx.Class.define("qx.ui.form.VirtualSelectBox",
       }, this);
     },
 
-    __applyHighlightStyle : function(value, old) {
-      var styles = [];
-      if (value != null) {
-        var keys = Object.keys(value);
-        var k;
-        for (k=0; k< keys.length; k++) {
+    __getHighlightStyle : function() {
+      var highlightAppearance =
+        qx.theme.manager.Appearance.getInstance().styleFrom("list-search-highlight");
+      console.log('__getHighlightStyle=', highlightAppearance);
```

Comment:
Is this still there? Should be removed before merging.

Reply:
Thanks for the catch. Removed.